src/pages/images/CachedImageList.tsx

```diff
@@ -39,14 +44,24 @@ const ImageList: FC = () => {
 
   const headers = [
     { content: "Name", sortKey: "name" },
-    { content: "Architecture", sortKey: "architecture" },
-    { content: "Type", sortKey: "type" },
-    { content: "Upload date", sortKey: "uploaded_at" },
-    { content: "Size", sortKey: "size" },
+    {
+      content: "Architecture",
+      sortKey: "architecture",
+      className: "architecture",
+    },
+    { content: "Type", sortKey: "type", className: "type" },
+    {
+      content: "Upload date",
+      sortKey: "uploaded_at",
+      className: "uploaded_at",
+    },
+    { content: "Size", sortKey: "size", className: "u-align--right size" },
     { "aria-label": "Actions", className: "actions" },
   ];
 
-  const rows = images.map((image) => {
+  const filteredImages = images;
+
+  const rows = filteredImages.map((image) => {
```

Comment:
Should filter based on the `query` here. Currently the query is set but not used to filter.

Reply:
Whoops, that is fixed now.